lib/bolt/target.rb

```diff
@@ -72,6 +73,11 @@ def protocol
       @uri_obj.scheme || @protocol
     end
 
+    # path is meant to be a temporary property that describes a file path on the target
```

Comment:
Should this be part of this PR since nothing's using it?

Reply:
Oh yeah, I'll drop this out. Was a separate experiment.

Reply:
Updated and rebased.